packages/web-react/src/components/RadioField/README.md

```diff
@@ -16,6 +16,7 @@ Use RadioField when you have a group of mutually exclusive choices and only one
 | `value`         | string  | Input value                    |
 | `isDisabled`    | boolean | Whether is field disabled      |
 | `isChecked`     | boolean | Whether is field checked       |
+| `isItem`        | boolean | Whether is an Item             |
```

Comment:
Can we describe it better? I do not understand what I should imagine under `Item`. Is it that another component is used and it is displayed differently?

Reply:
Saying something like _render in Item mode_ is the minimum that works for me at least in the jsdoc comments. Here it could/should be described even better, in more detail.

I even think it deserves its own section in README. From the design point of view, the Item mode is handled in a specific way. Then the API description can be brief.

Reply:
added, check it, please